net/xh/Portfile

```diff
@@ -0,0 +1,321 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo  1.0
+
+github.setup        ducaale xh 0.9.2 v
+revision            0
+
+categories          net
+platforms           darwin
+license             MIT
+maintainers         {outlook.de:judaew @judaew} openmaintainer
+
+description         xh is a friendly and fast tool for sending HTTP requests.
+long_description    ${description} It reimplements as much as possible of \
+                    HTTPie's excellent design.
+homepage            https://github.com/ducaale/xh
+
+checksums           xh-0.9.2.tar.gz \
```

Comment:
Don't repeat yourself... in this case you're repeating the version.

If `checksums ${distname}${extract.suffix}` does not work here, at least use `xh-${version}${extract.suffix}` instead.